docs/standard/assembly/contents.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Assembly contents
+description: A static assembly can consist the assembly manifest containing assembly metadata, type metadata, code that implements types, and a set of resources.
```

Comment:
Everything else looks good, except for this one. The description doesn't make sense to me - could you please reword it?

Reply:
@IEvangelist  - How about this version: 

This article describes the contents of a .NET assembly, which can include assembly metadata, type metadata, MSIL code, and resources.

I would have just said that the article describes the contents, but that was way under the 100 char min.

Reply:
Sounds good :shipit: